p5js/FeatureExtractor/FeatureExtractor_Image_Classification/sketch.js

```diff
@@ -80,6 +81,18 @@ function setupButtons() {
   // Predict Button
   buttonPredict = select('#buttonPredict');
   buttonPredict.mousePressed(classify);
+
+  // Save model
+  saveBtn = select('#save');
+  saveBtn.mousePressed(function() {
+    classifier.save();
+  });
+
+  // Load model
+  loadBtn = select('#load');
```

Comment:
I wonder if we can use the file input p5 element? (See: https://p5js.org/reference/#/p5/createFileInput) Not sure if `select()` compatible and will know it's a file input.

Reply:
I tried with `createFileInput()` but you also need to setup the position and it became a bit messy with the structure we have so far. But perhaps I'm not using it in the right way.